html/elements/track.json

```diff
@@ -0,0 +1,482 @@
+              },
+              {
+                "version_added": "24",
+                "flag": {
+                  "type": "preference",
+                  "name": "media.webvtt.enabled",
+                  "value_to_set": "true"
+                }
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "12.1"
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": "6"
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "default": {
+          "__compat": {
+            "support": {
+              "webview_android": {
+                "version_added": "25"
+              },
+              "chrome": {
+                "version_added": "23"
+              },
+              "chrome_android": {
+                "version_added": "25"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": [
+                {
+                  "version_added": "31"
+                },
+                {
+                  "version_added": "24",
+                  "flag": {
+                    "type": "preference",
+                    "name": "media.webvtt.enabled",
+                    "value_to_set": "true"
+                  }
+                }
+              ],
+              "firefox_android": [
+                {
+                  "version_added": "31"
+                },
+                {
+                  "version_added": "24",
+                  "flag": {
+                    "type": "preference",
+                    "name": "media.webvtt.enabled",
+                    "value_to_set": "true"
+                  }
+                }
+              ],
+              "ie": {
+                "version_added": "10"
+              },
+              "ie_mobile": {
+                "version_added": null
+              },
+              "opera": {
+                "version_added": "12.1"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "6"
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "kind": {
```

Comment:
I had no idea either. As all the support data was either "false" or "null", I removed it. When 1 browser will support this (whatever it is), it will come back ;-)

Reply:
Makes sense. Thanks!